src/lib/utils.js

```diff
@@ -34,7 +34,7 @@ utils.handleMergeArrays = function (a, b) {
 // merge atomizer configs into a single config
 utils.mergeConfigs = function (configs/*:Config[]*/)/*:Config*/ {
     // TODO: Offer option to warn on conflicts
-    return _.merge.apply(null, configs.concat(utils.handleMergeArrays));
+    return _.mergeWith.apply(null, configs.concat(utils.handleMergeArrays));
```

Comment:
@roderickhsiao Any particular reason to make this change, since we're not using the customer argument that differentiates `mergeWith` from `merge`?